www/less/misc.less

```diff
@@ -53,6 +53,29 @@ a{
     padding-bottom:0px;
 }
 
+// Horizontal divider between title/description and lead section text
+#content_block_0_hr {
+    border:1px solid #999;
```

Comment:
@montehurd hey, we need the hairline border here. it's important that it's 0.5px on retina screens. can we try  transform: scaleY(0.5); 

more info here http://atirip.com/2013/09/22/yes-we-can-do-fraction-of-a-pixel/

Also, i checked the color on my phone screen again and i think we should change it to #CCC 
Final screenshot

![image](https://cloud.githubusercontent.com/assets/1161093/12039109/a0b230f2-ae2b-11e5-8f7e-a32af9a17dc5.png)